drizzlepac/haputils/analyze.py

```diff
@@ -303,7 +344,7 @@ def analyze_data(input_file_list, log_level=logutil.logging.NOTSET):
         # for all the good data.
         split_sfilter = sfilter.upper().split('_')
         for item in split_sfilter:
-            if item.startswith(('G', 'PR')) and not is_zero:
+            if item.startswith(('G', 'PR')) and not is_zero and type.upper() != "MVM":
```

Comment:
It is not clear to me what message would be generated for Grism or Prism data, if any, as it seems that if `type == MVM`, the image would not be flagged to be skipped at all with this change.  This change would leave, if I understand correctly, Grism/Prism data with `no_proc_key`= None, or in other words, it would still be processed.

Reply:
The logic here is correct, but I make the if-elif more elaborate in order to accommodate additional log messages describing the action.  Also add more comments in commit 111c243.

Reply:
The extra log messages are perfect!